src/uikit.cs

```diff
@@ -15492,6 +15569,53 @@ public interface UIPressesEvent {
 		NSSet<UIPress> GetPresses (UIGestureRecognizer gesture);
 	}
 
+	[NoWatch, NoTV, iOS (10,0)]
+	[BaseType (typeof(NSObject), Delegates=new string [] {"Delegate"}, Events=new Type [] { typeof (UIPreviewInteractionDelegate)})]
+	[DisableDefaultCtor]
+	interface UIPreviewInteraction {
+
+		[Export ("initWithView:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (UIView view);
+
+		[NullAllowed, Export ("view", ArgumentSemantic.Weak)]
+		UIView View { get; }
+
+		[NullAllowed, Export ("delegate", ArgumentSemantic.Weak)]
+		IUIPreviewInteractionDelegate Delegate { get; set; }
+
+		[Export ("locationInCoordinateSpace:")]
+		CGPoint GetLocationInCoordinateSpace ([NullAllowed] UICoordinateSpace coordinateSpace);
+
+		[Export ("cancelInteraction")]
+		void CancelInteraction ();
+	}
+
+	interface IUIPreviewInteractionDelegate { }
+
+	[NoWatch, NoTV, iOS (10, 0)]
+	[Protocol, Model]
+	[BaseType (typeof(NSObject))]
+	interface UIPreviewInteractionDelegate {
+
+		[Abstract]
+		[Export ("previewInteraction:didUpdatePreviewTransition:ended:")]
+		[EventArgs ("NSPreviewInteractionPreviewUpdate")]
+		void DidUpdatePreviewTransition (UIPreviewInteraction previewInteraction, nfloat transitionProgress, bool ended);
+
+		[Abstract]
+		[Export ("previewInteractionDidCancel:")]
+		void DidCancel (UIPreviewInteraction previewInteraction);
+
+		[Export ("previewInteractionShouldBegin:")]
+		[DelegateName ("Func<UIPreviewInteraction,bool>"), DefaultValue(true)]
+		bool ShouldBegin (UIPreviewInteraction previewInteraction);
+
+		[Export ("previewInteraction:didUpdateCommitTransition:ended:")]
+		[EventArgs ("NSPreviewInteractionPreviewUpdate")]
```

Comment:
For some reason this won't show up here `[EventArgs ("NSPreviewInteractionPre` but in the file it is complete